File: HealthCatalyst.Models/Address.cs
namespace HealthCatalyst.Models
{
    /// <summary>
    /// Implementation of Address
    /// 
    /// For simplicity of demo assume US address
    /// </summary>
    public class Address
    {
        public int AddressId { get; set; }
        public int? PersonId { get; set; }
        public string Line1 { get; set; }
        public string Line2 { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
    }
}

File: HealthCatalyst.Web/ClientApp/src/app/models/address.ts
import {IAddress} from '../interfaces/iaddress'

export class Address implements IAddress{
    constructor(addressId: number, personId: number, line1: string, line2 : string, city: string, state: string, postalCode : string){
      this.addressId = addressId;
      this.personId = personId;
      this.line1 = line1;
      this.line2 = line2;
      this.city = city;
      this.state = state;
      this.postalCode = postalCode;
    }

    addressId: number;
    personId: number;
    line1: string;
    line2: string;
    city: string;
    state: string;
    postalCode: string;

  toString(): string {
    return `${this.line1}<br/>${this.city} ${this.state} ${this.postalCode}`;
  }
}
